plugins/woocommerce/client/blocks/assets/js/base/stores/woocommerce/cart.ts

```
@@ -1,701 +1,749 @@
+                    const batchResponse = yield fetch(`${state.restUrl}wc/store/v1/batch?intent=${intent}`, {
+                        method: "POST",
+                        headers: { Nonce: state.nonce, "Content-Type": "application/json" },
+                        body: JSON.stringify({ requests: batchRequests }),
+                    });
+
+                    const data = yield batchResponse.json();
+
+                    if (isApiErrorResponse(batchResponse, data)) throw generateError(data);
+
+                    if (_currentId !== _requestCounter) {
+                        currentDeletes.forEach((k) => deleteQueue.add(k));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)